sdk/servicebus/Azure.Messaging.ServiceBus/src/ServiceBusConnectionStringBuilder.cs

```diff
@@ -1,449 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-namespace Azure.Messaging.ServiceBus
-{
-    using System;
-    using System.Collections.Generic;
-    using System.Globalization;
-    using System.Text;
-    using Primitives;
-
-    /// <summary>
-    /// Used to generate Service Bus connection strings.
-    /// </summary>
-    internal class ServiceBusConnectionStringBuilder
```

Comment:
It looks like there are some settings that were parsed here, that aren't handled in ConnectionStringParser:
OperationTimeout, TransportType

I wonder if we should be respecting these values, at least if no ServiceBusClientOptions is passed in to the ctor, since these could also be set there. I think OperationTimeout corresponds to RetryOptions.TryTimeout.

Reply:
Users can set OperationTimeout and TransportType in `ServiceBusClientOptions`

>It looks like there are some settings that were parsed here, that aren't handled in ConnectionStringParser:
OperationTimeout, TransportType

It needs more discussion whether we need to add OperationTimeout, TransportType in ConnectionStringParser or not.